src/main/java/lotto/model/LottoPrizeRank.java

```diff
@@ -0,0 +1,35 @@
+package lotto.model;
+
+
+import java.util.stream.Stream;
+
+public enum LottoPrizeRank {
+    NONE(0, 0),
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000);
```

Comment:
```suggestion
    SIX(6, 2_000_000_000);
```

숫자는 '_' 을 같이 사용하면 가독성을 높힐 수 있을 것 같아요 :)

Reply:
너무 좋습니다!! 바로 반영해야겠어요!! 알고 있지만 습관이 잘 안잡힌 부분인 것 같은데 언급해주셔서 감사드려요 🙏